app/models/manageiq/providers/openstack/cloud_manager/host_aggregate.rb

```diff
@@ -1,8 +1,118 @@
 class ManageIQ::Providers::Openstack::CloudManager::HostAggregate < ::HostAggregate
+  include SupportsFeatureMixin
```

Comment:
Makes sense. I was probably following another (perhaps bad) example, but yeah removing is right here.